backend/setup/wsgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'setup.settings')
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'myproject.settings')
```

Comment:
it should be setup.settings since myproject folder was changed to setup folder - but it doesn't seem to impact the running app